.github/workflows/ci.yml

```diff
@@ -74,7 +74,8 @@ jobs:
       env:
         COLOR: 'yes'
       run: |
-        pytest tests --cov-report xml
+        pytest tests
+        python -m coverage xml
```

Comment:
@webknjaz Not sure if this makes sense to you, but seems that if I use --cov-report, then codecov misses the `__init__.py` files (although they appear in the terminal coverage output).

Reply:
Dunno. It might depend on the args that `pytest.ini` has, or env vars...

Reply:
I don't see any "skip empty" tho

Reply:
It's not empty, it contains half the project. :P

See codecov summary above, it's 119 lines difference.

Reply:
Suppose I mean "skip covered"